apps/code-server/template/before.sh.erb

```diff
@@ -1,5 +1,8 @@
-source "<%= session.staged_root %>/common_before.sh"
+<%= ERB.new(File.read('../common_files/ood_session.sh.erb'), eoutvar: 'child').result(binding) %>
+<%= ERB.new(File.read('../common_files/work_directory.sh.erb'), eoutvar: 'child').result(binding) %>
+<%= ERB.new(File.read('../common_files/job_environment.sh.erb'), eoutvar: 'child').result(binding) %>
+<%= ERB.new(File.read('../common_files/sha1_password.sh.erb'), eoutvar: 'child').result(binding) %>
+
+export OOD_APP_NAME="VS Code Server"
```

Comment:
I wonder if we can't get this information from the form directly so it's defined only once.

Reply:
I was also expecting the app name to be available in the template objects provided by OoD (https://osc.github.io/ood-documentation/latest/how-tos/app-development/interactive/template.html#erb-objects). But I checked `session` and `context` and those do not contain it.